### ISSUE TITLE
jQuery dialog boxes

### ISSUE THREAD

**Issue by nickburne** [pootlepress/pootle-page-builder#30]
1. prefix jq ui with ppb
2. create custom dialogs like SO

**Comment by nickburne** [comment]
To confirm, when you do this do it on a separate branch. This is 0.3.0 so doesn't have to be done yet.

**Comment by shramee** [comment]
Done, we had to do for tabs too... Our panels use dialog and tabs, lemme know how it works for ya ;)

**Comment by shramee** [comment]
Branch ppbDialog ;)

**Comment by nickburne** [comment]
Looking good, to confirm this is for  "2. create custom dialogs like SO"

Correct?

**Comment by shramee** [comment]
This ( ppbDialog ) is custom jquery widget. Extended from jquery ui, major changes include, overlay managed by widget ( each and every instance of dialog had this code so we moved it to widget's core functionality to reduce clutter ) and removed all ui- classes.
We also got our custom tabs widget ( ppbTabs ) to get rid of more 'ui-' classes...
This is all done for reducing influece of jq ui css on our ui elements.
